recipe/meta.yaml

```diff
@@ -17,40 +16,60 @@ build:
 
 requirements:
   host:
-    - python >=3.6,<3.10
-    - poetry
+    - python >=3.6.2,<3.11
+    - poetry-core >=1.0.0
     - pip
   run:
-    - python >=3.6,<3.10
-    - boto3 >=1.16.8,<2.0.0
-    - botocore >=1.19.8,<2.0.0
-    - numpy >=1.18.0,<2.0.0
-    - pandas >=1.1.0,<2.0.0
-    - pyarrow >=2.0.0,<5.1.0
-    - redshift_connector ~=2.0.887
+    - boto3 >=1.20.17,<2.0.0
+    - botocore >=1.23.17,<2.0.0
+    - jsonpath-ng >=1.5.3,<2.0.0
+    - numpy >=1.18.0,<1.22
+    # Conda should choose the correct numpy versions for the different python versions.
+    - openpyxl >=3.0.0,<3.1.0
+    - opensearch-py >=1.0.0,<2.0.0
+    - pandas >=1.1.0,<1.4.0
+    # Conda should choose the correct pandas versions for the different python versions.
+    - pg8000 >=1.16.0,<1.23.0
+    - progressbar2 >=3.53.3,<4.0.0
+    - pyarrow >=2.0.0,<6.1.0
     - pymysql >=0.9.0,<1.1.0
-    - pg8000 >=1.16.0,<1.22.0
-    - openpyxl ~=3.0.0
+    - python >=3.6.2,<3.11
+    - redshift_connector >=2.0.889,<2.1.0
+    - requests-aws4auth >=1.1.1,<2.0.0
+    - xlrd >=2.0.1,<3.0.0
+    - xlwt >=1.3.0,<2.0.0
```

Comment:
I have included these as the package can otherwise not be noarch. From what I can see in the docs they can also safely be installed under the newer python versions. Do you have any other input/experience here?

Reply:
These two libraries are only required on Py3.6, however as you point out they allow any Python version in their requirements so should be fine to keep them